cmd/api/main.go

```diff
@@ -21,6 +21,8 @@ type Book struct {
 	Name      string    `json:"name"`
 	Price     *float32  `json:"price"`
 	Inventory *int      `json:"inventory"`
+	CreatedAt string    `json:"created_at"`
```

Comment:
Do we want to use/show the information of date/time inside our api code or it is just do order the list that db returns?
In other words: Should we created fields for it in our Book struct or not? This will change the way we make tests.

Reply:
I would say its OK to show the timestamps on the API, even good to have it.
Lets have it in the API too.

Reply:
That said, you could just hide it in the JSON data with an explicit json ignore tag, like this `json:"-"`.